korge-core/test/korlibs/image/bitmap/BitmapSliceTest.kt

```diff
@@ -27,6 +27,25 @@ class BitmapSliceTest {
         assertEquals("Rectangle(x=24, y=24, width=24, height=24)", bmp.sliceWithSize(16, 16, 32, 32).sliceWithSize(8, 8, 40, 40).bounds.toString())
     }
 
+    @Test
+    fun testPutSliceWithBorder() {
```

Comment:
Regarding test. Maybe the Bitmap32 can be converted into a Bitmap1/Bitmap8 and then: https://github.com/korlibs/korge/blob/11993ae7828c96ee261b5d610904c8ca5430b8d2/korge-core/test/korlibs/image/bitmap/Bitmap1Test.kt#L8

https://github.com/korlibs/korge/blob/65ca3e1e284fbb4bffb413a1ba687437befef4bc/korge-core/src/korlibs/image/bitmap/Bitmap8.kt#L49

`bmp.tryToExactBitmap8().toLines(".*#").joinToString("\n")`

or use a reference image. Since we are in korge-core, we should be able to generate screenshots.

Reply:
I checked using Bitmap1 in the unit test but `putWithBorder()` is not implemented for Bitmap1. What seems to be missing is the implementation of `slice()` for Bitmap1.

Reply:
Since `Slice()` is not implemented for Bitmap1 I would like to keep the unit test as it is now.

Reply:
I can try with Bitmap8 indeed and create an imput and output image with indexes...